bot/exts/fun/connect_four.py

```diff
@@ -426,7 +424,6 @@ async def connect_four(
 
         await self._play_game(ctx, user, board_size, str(emoji1), str(emoji2))
 
-    @guild_only()
```

Comment:
Why can this be removed? The code that now handles DM-only checks is in the whitelist decorators, but that isn't applied here.

Reply:
The whitelist decorator is applied globally here:
https://github.com/python-discord/sir-lancebot/blob/377dcf969c74ef83cf534e7ba058fbb87bd28d88/bot/__main__.py#L10